### PR TITLE
Add package autodiscovery

### PR DESCRIPTION
**Pull request by imjohnbon**
Added package autodiscovery for Laravel 5.5.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     ],
     "license": "MIT",
     "require": {
-        "laravel/framework": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*"
+        "laravel/framework": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*|5.5.*"
```
Comment (mattstauffer) [inline composer.json]: Have we tested it on 5.5 yet?

Comment (imjohnbon) [inline composer.json]: @mattstauffer I'd actually love your opinion on that given the circumstances. We can't fully be sure until Laravel 5.5 is officially released (we could test off the latest dev version of Laravel of course, but there could be issues that come from it being dev). Given that package auto-discovery doesn't exist before 5.5, should I just re-PR this once Laravel 5.5 releases and we can fully test it on a stable release? Or is there any value in doing testing right now while 5.5 is still in dev?

Comment (mattstauffer) [inline composer.json]: @imjohnbon I would do package auto discovery as one PR. We merge it. Then do another PR. we test it against 5.5. If it works, I think we merge that too, then test again at launch.

Comment (imjohnbon) [inline composer.json]: @mattstauffer Good call, I removed the 5.5 upgrade and just left the package autodiscovery for this PR.
